eclair-core/src/main/resources/reference.conf

```diff
@@ -109,6 +109,7 @@ eclair {
   payment-request-expiry = 1 hour // default expiry for payment requests generated by this node
   multi-part-payment-expiry = 60 seconds // default expiry for receiving all parts of a multi-part payment
   min-funding-satoshis = 100000
+  max-funding-satoshis = 16777216 // to open channels larger than 16777216 you must enable the large_channel_support feature in 'eclair.features'
```

Comment:
```suggestion
  max-funding-satoshis = 16777215 // to open channels larger than 16777216 you must enable the large_channel_support feature in 'eclair.features'
```

Off-by-one error?

Reply:
Good catch! We wanted the max-funding to be equal to the channel limit which is 2^24 but when validating the parameters we make sure the channel size is LTE than the max-funding. Thus a user with wumbo **disabled** can open up to `16777215`, with wumbo **enabled** up to `16777216` by default because of the max-funding limit.